config/locales/to_change_everything/th.yml

```diff
@@ -0,0 +1,139 @@
+ภาษาไทย:
+  layouts:
+    to_change_everything:
+      language_direction: ltr
+      lang_code: th
+      video_url: https://player.vimeo.com/video/111159930
+      head:
+        title: "เพื่อเปลี่ยนแปลงทุกสิ่ง คำวิงวอนจากผู้นิยมลัทธิอนาธิปไตย / CrimethInc. Ex-Workers' Collective"
+        meta:
+          title: เพื่อเปลี่ยนแปลงทุกสิ่ง คำวิงวอนจากผู้นิยมลัทธิอนาธิปไตย
+          description: The case for complete self-determination—a guide for the furious, the curious, and the pure of heart
+          url: http://tochangeeverything.com/ภาษาไทย
```

Comment:
@veganstraightedge I am not not sure where these routes get set up

Reply:
Nowhere! 🎉

`tochangeeverything.com` is also mapped to the Heroku app.

Then a Rack middleware redirects all other domains to `crimethinc.com` domain, adding any path suffix needed. In this case, `tochangeeverything.com/:etc` redirects to `crimethinc.com/tce/:etc`.

https://github.com/crimethinc/website/blob/cff50d6627cce5bd0ca8e4e0bef05ebbf2d4f565/app/middlewares/rack/domain_redirect.rb#L6